lib/secretFactory.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const BaseFactory = require('./baseFactory');
+const Secret = require('./secret');
+const iron = require('iron');
+const nodeify = require('./nodeify');
+const hoek = require('hoek');
+// Get symbols for private fields
+const password = Symbol();
+
+let instance;
+
+/**
+ * Seal secret value and return the secret with the sealed value
+ * @method sealSecret
+ * @param  {SecretModel}        secret  Secret to seal
```

Comment:
Or at least not based on how you're using it: https://github.com/screwdriver-cd/models/pull/92/files#diff-db62f6a8d7d20845993eb9a470a318dfR81